app/assets/stylesheets/form.scss

```diff
@@ -0,0 +1,55 @@
+form.like-form{
```

Comment:
Okay, so, as i had to change the whole website in order to use react, i removed a bunch of stuff, that, right now i don't have, and i will have to put back, but i can't do it now, because i can't see that page in the website, this is supposed to be seen in the home page, which you can get there, only if you properly signed up or logged in.

   As i cannot do that yet, because we still don't get to that part, i think it's better for both of us to finish this PR, follow along with the log in PR, and then, do the redirecting PR that we talk about last time, in order to improve this.